### ISSUE TITLE
Jupyter Lab 2.1.5 fails to build after installing higlass-jupyter

### ISSUE THREAD

**Issue by CriticalSci** [higlass/higlass-python#63]
The system and dependencies:
Mac OS 10.14.6
Python3, node.js and MacFuse installed via Homebrew.
higlass-python (0.4.0) and jupyterlab installed via pip3.

The JupyterLab setup:
JupyterLab v2.1.5
Known labextensions:
   app dir: /usr/local/share/jupyter/lab
        @jupyter-widgets/jupyterlab-manager v2.0.0  enabled  OK
        jupyter-matplotlib v0.7.2  enabled  OK

After running `jupyter labextension install higlass-jupyter`
I get the following error:
```
Building jupyterlab assets (build:prod:minimize)
An error occured.
RuntimeError: JupyterLab failed to build
```
Here is the [error log](https://github.com/higlass/higlass-python/files/4825095/jupyterlab-debug-ddcuq4gz.log). Any help would be greatly appreciated! Cheers.

**Comment by flekschas** [comment]
Thanks for reporting. This might be related to `@jupyter-widgets/base` which was updated to `v3` recently.

The relevant build error seems to be `ModuleNotFoundError: Module not found: Error: Can't resolve 'pixi.js' in '/usr/local/share/jupyter/lab/staging/node_modules/higlass/dist'`

The easiest fix, for now, is to downgrade Jupiter lab until we find a fix.

**Comment by CriticalSci** [comment]
Thanks Fritz.
I tried downgrading Jupyter Lab to 2.0.0 and 1.2.16 but they are still failing to build with the same module error after attempting to install higlass-jupyter.
Matplotlib widget for example installs and works fine though in both.

It seems pixi.js is definitely missing in the node_modules directory after higlass install.

**Comment by flekschas** [comment]
I think the issue might be caused by Jupyter lab v2, which I think bumped the version of `@jupyter-widgets/base` from `v2` to `v3`. I will try to investigate in the next days.

If you are flexible regarding the versions, I would start from the https://github.com/higlass/scipy19/blob/master/environment.yml. In this file we fixed the version numbers that definitely worked. But again my guess is that higlass-python is not yet compatible with jupyter lab v2.

On a side note, I don't think this has anything to do with pixi.js. The jupyter lab extension system is, unfortunately, very brittle and can break with any new version without providing really useful debugging information.

**Comment by flekschas** [comment]
Okay this is an issue with [higlass](https://github.com/higlass/higlass) itself. Since the hglib version in `package.json` is specified as `^1.8.1` the newest version of hglib will be installed (i.e., `1.9.4` at the time of this writing). I traced the error and found out that in `v1.9.2` an error was introduced which causes the following error during the build (`npm run build` process):

```
Module not found: Error: Can't resolve 'pixi.js' in '/Users/Fritz/Code/4DN/higlass-python/js/node_modules/higlass/dist'
```

@pkerpedjiev Any idea what changed in that version of hglib?

A temporary fix is to fix the version to `v1.9.1` but I wonder if we should rather fix hglib.

**Comment by flekschas** [comment]
It turns out there's [another issue](https://github.com/higlass/simple-httpfs/issues/9) at the moment related to simple-httpfs. Once the two are fixed, higlass-python should be installable again.
